### DISCUSSION TITLE
nvim-treesitter-textobjects upstream bug causing startup error, disabling it in disabled.lua has no effect

### DISCUSSION THREAD

**Discussion by ssh352** [LazyVim/LazyVim#3203] [Q&A]
https://github.com/nvim-treesitter/nvim-treesitter-textobjects/issues/617

how to properly eliminate it from loading? I also tried uninstall it, but it gets installed automatically again.

**Comment by dpetka2001** [comment]
Revert the `nvim-treesitter` to commit `19ac9e8` from this morning. It will eventually be fixed.

**Comment by surmish** [comment]
Bdw, I was able to get over this startup error, by compiling `neovim` latest (`0.11.xx`) manually with latest `libtree-sitter.so.0.22` library and `tree-sitter 0.22.6` executable.

I think compiling `neovim` with bundled tools resulted in this startup error for me.

**Reply by dpetka2001** [reply]
Error is gone with latest `nvim-treesitter` update.

**Reply by ssh352** [reply]
all good now, thanks everyone
